llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -17194,12 +17194,56 @@ static bool combine_CC(SDValue &LHS, SDValue &RHS, SDValue &CC, const SDLoc &DL,
     return true;
   }
 
+  // If XOR is reused and has an immediate that will fit in XORI,
+  // do not fold.
+  auto IsXorImmediate = [](const SDValue &Op) -> bool {
```

Comment:
lambda functions should following the normal function naming convention, which starts with lower case. ditto for other occurrences here.